src/agent/structures/structures.ts

```diff
@@ -73,6 +79,7 @@ export interface ListArchivesParameters {
   page_id?: string;
   sort_order?: SortOrder;
   limit?: number;
+  highlights: Highlights;
```

Comment:
```suggestion
  highlights?: Highlights;
```
It should be optional like in newer versions

Reply:
OK in v3.5